src/services/splitters.rs

```diff
@@ -230,6 +230,7 @@ pub fn split_slots_into_1h_slots(slots: Vec<Slot>) -> Vec<Slot> {
 ///         Task `Sleep` 00 - 06 (next day)
 /// ```
 pub fn split_crossed_tasks(tasks: &mut Vec<Task>) {
```

Comment:
@moaz-mokhtar 
Can you rename `split_crossed_tasks` to `split_tasks_on_midnight` - or something similar - so the intention is clear?  
Nobody knows what a 'crossed_task` is just by reading these words.

Reply:
Ok